interface/patient_tracker/patient_tracker_status.php

```diff
@@ -116,8 +116,8 @@
             </div>
             <div class="position-override">
                 <div class="btn-group oe-opt-btn-group-pinch" role="group">
```

Comment:
So, such as those, Are they can remove?

Reply:
yep, would consider adding btn-group-pinch and oe-opt-btn-group-pinch to the removal (assuming that @tywrenn agrees)